Old Bailey Online.js

```diff
@@ -176,23 +176,21 @@ function doWeb(doc, url) {
 var testCases = [
 	{
 		"type": "web",
-		"url": "http://www.oldbaileyonline.org/browse.jsp?id=t16780828-12&div=t16780828-12&terms=hog#highlight",
+		"url": "https://www.oldbaileyonline.org/browse.jsp?id=t16780828-12&div=t16780828-12&terms=hog#highlight",
 		"items": [
 			{
 				"itemType": "case",
+				"caseName": "Theft > animal theft, 28th August 1678.",
 				"creators": [],
-				"notes": [],
+				"extra": "Reference Number: t16780828-12",
```

Comment:
It seems that this id consists of a letter `t`, the date `16780828`=1678-08-28 and a counter `-12`. [It is written that the reference number is](https://www.oldbaileyonline.org/static/HowToReadTrial.jsp)

> a unique reference number, which can be used to search for the trial subsequently and should be used when citing the trial in any written work

Thus, it looks that this may fit into `docketNumber` where there are some changes that it is used in citations as suggested. Also it is nothing original but a number given by this database. I will then change this...